src/pudl/output/ferc714.py

```diff
@@ -530,8 +530,10 @@ def georef_counties(self, update=False):
         analyses.
         """
         if update or self._counties_gdf is None:
-            census_counties = pudl.analysis.service_territory.get_census2010_gdf(
-                pudl_settings=self.pudl_settings, layer="county", ds=self.pudl_out.ds)
+            census_counties = pudl.output.censusdp1tract.get_layer(
+                "county",
```

Comment:
can you add `layer=` in these so it is clear?

Reply:
Done.